spanner/tests/unit/test_snapshot.py

```diff
@@ -417,6 +417,8 @@ def _execute_sql_helper(
             partition_token=partition,
             seqno=sql_count,
             metadata=[('google-cloud-resource-prefix', database.name)],
+            timeout=google.api_core.gapic_v1.method.DEFAULT,
+            retry=google.api_core.gapic_v1.method.DEFAULT
```

Comment:
This helper should probably grow support for passing explicit `timeout` / `retry` arguments;  we would then need a driver testcase (or two) which exercises that.

Reply:
80e22369b5fe1d5ddb6a9f00a0663e5acd7ac6f2